src/PointwiseFunctions/AnalyticData/WaveEquation/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY WaveEquationAnalyticData)
+
+set(LIBRARY_SOURCES
+  RegularSphericalWaveKerrSchild.cpp
+  )
+
+add_spectre_library(${LIBRARY} ${LIBRARY_SOURCES})
+
+target_link_libraries(
+  ${LIBRARY}
+  INTERFACE DataStructures
+  INTERFACE ErrorHandling
+  INTERFACE GeneralRelativity
```

Comment:
Spectral, Parallel, Utilities

Reply:
Included Spectral and Utilities, but which library do you mean by `Parallel`? Also, why does this need to link with Spectral?